src/Illuminate/Foundation/Application.php

```diff
@@ -245,6 +245,22 @@ public static function configure(?string $basePath = null)
             ->withProviders();
     }
 
+    /**
+     * Get the application from the bootstrap file.
+     *
+     * @param  string  $bootstrapPath
+     * @return static
+     */
+    public static function fromBootstrap(string $bootstrapPath)
+    {
+        $application = (require_once $bootstrapPath);
+
+        return match (true) {
+            ($application instanceof static) => $application,
+            default => null,
+        };
```

Comment:
A `match(true)` for just two cases seems a little overkill to me

Reply:
I looked at `inferBasePath()`, which also used a `match(true)` for just two cases.

Reply:
Well yeah, true... 🙂